### DISCUSSION TITLE
New component to show date/time, e.g. v-datetime

### DISCUSSION THREAD

**Discussion by t7tran** [directus/directus#6104] [Feature Requests]
There is a `datetime` display component for rendering date/time with localization support during `date-fns`. It also has the sophisticate mechanism of lazy loading locales. It would be great to have the consistent behaviour in the extensions, e.g. custom modules.

Is it an overkill to have it registered as a v-datetime component? or is there a better way to format date time in the same manner in our custom code?

**Comment by rijkvanzanten** [comment]
Makes sense to me! Wanna open a PR to rework that display into a regular `v-datetime` component? 🙂

**Reply by t7tran** [reply]
Sure, will do.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
